### PR TITLE
Add redesigned homepage

### PR DESCRIPTION
**Pull request by zackkrida**
## Fixes

Fixes #592

### Todo

- [x] Cycle through search terms and images
- [x] Proper text formatting and logo mark
- [ ] Hook up search form
- [ ] Content switcher inside of search form
- [ ] i18n
- [ ] Tests? Maybe?

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
Referencing work in PR #603 that might be helpful when printing attribution for these images: https://github.com/WordPress/openverse-frontend/pull/603/files#diff-753722b3502b1d9b799f6793aa6c19cbd061238bd86e78044fc7e865a220734eR40

**Comment by panchovm** [comment]
Adding two notes here from #613 
1. With the images transition, we need to add a typing effect on the placeholder to highlight the search + results effect desired.
2. The content switcher is smaller than the original one. It should be the same, without any change.
3. The content type items placed in the switcher popover should follow the change request in #639 

From this PR, and [Dhruv's comment](https://github.com/WordPress/openverse-frontend/discussions/592#discussioncomment-1996695), I would add the following:
1. Apply the mobile gallery to breakpoints `sm` and `md`.
2. As the point above, in `md` openverse logo should be placed on the top, like on mobile.
3. Images size is `120px`.
4. Spacing between images is `16px`.
5. Description about openverse content should be text-align left

**Comment by dhruvkb** [comment]
@panchovm I've addressed the second half of your feedback, and am working on the points in the upper half. Please re-review, thanks!

**Comment by dhruvkb** [comment]
@panchovm to speed up the process and parallelise things, I'll make a separate issue for fixing differences between the content switcher inside the search bar and the Figma mockups.

Except for this, the homepage is ready for review.

**Comment by panchovm** [comment]
The transition looks awesome ✨ I noticed the following:
1. On the horizontal arrangement of images there is a scroll. We should disable it and show images as the viewport increases its width.
2. The caption text about openverse license is incorrect. It should say "All Openverse content is under a Creative Commons license or is in the public domain."
3. On `sm` and `md`, the title above search bar should be `Heading 3` style and say "Browse through over [number] million items to reuse"

Two notes aside:
* I noticed that on `md` search bar expands a lot, should we set a max-width value? I suggest a `700px` for the whole area (search input + CTA)
* The time-lapse between Universe, Indigenous Pottery, and Olympic games was inconsistent at first. But then I load it again and it was fixed. I don't know what might be happening.

Despite the above details, it looks marvelous. Excellent work.

**Review by zackkrida — COMMENTED** [review]
Great! One small suggestion but otherwise LGTM

**Comment by dhruvkb** [comment]
What would be the accessibility of absolutely positioning the rotating string above the input and making it ARIA-hidden (being purely presentational) at all times and invisible if some string is entered into the input field.

File: src/pages/index.vue [hunk 1]
```diff
@@ -1,23 +1,286 @@
...
+      contentType.value = type
+      contentSwitcher.value?.closeMenu()
+      await store.dispatch(`${SEARCH}/${UPDATE_QUERY}`, {
+        searchType: type,
+      })
+    }
+
+    const searchTerm = ref('')
+    const handleSearch = async () => {
+      await store.dispatch(`${SEARCH}/${UPDATE_QUERY}`, {
```
[the remark this answers is not in the document]
Comment (dhruvkb) [reply src/pages/index.vue:227]: Implemented in 2b690ad.

**Comment by dhruvkb** [comment]
Merged to parallelise fixes for known and potential bugs, sorry.

**Pull request merged**